ntoskrnl/mm/ARM3/kdbg.c

```diff
@@ -175,7 +175,7 @@ ExpKdbgExtPoolUsed(
 {
     ULONG Tag = 0;
     ULONG Mask = 0;
-    ULONG Flags = 0;
+    ULONG_PTR Flags = 0;
```

Comment:
Why is this ULONG_PTR? Flags don't usually change their size based on the architecture.

Reply:
`ULONG_PTR` is the required type by `KdbpGetHexNumber()`.
https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=KdbpGetHexNumber